google/auth/aws.py

```diff
@@ -349,35 +348,15 @@ class Credentials(external_account.Credentials):
     def __init__(
         self,
         audience,
-        subject_token_type,
```

Comment:
Is making this change backwards compatible?  I suspect it is not but I haven't done a refactor like this before.

Reply:
Actually, most of the changes I've made are backwards compatible.  All I've done is instead of passing the parameters through explicitly, they're being passed through via kwargs, but they're still being passed through.  The only difference is with the from_info in aws.py. It will pass a workforce_pool_user_project=None through, but this shouldn't make a difference, because aws.Credentials would initialize external_account.Credentials, which has workforce_pool_user_project=None as the default value

Reply:
What if the arguments were positional? Do we need to pass `*args` as well?